app/uk/gov/hmrc/plasticpackagingtax/registration/views/components/Styles.scala

```diff
@@ -18,7 +18,8 @@ package uk.gov.hmrc.plasticpackagingtax.registration.views.components
 
 object Styles {
 
-  val gdsPageLegend  = "govuk-fieldset__legend--xl"
-  val gdsPageHeading = "govuk-heading-xl"
-  val gdsPageLabel   = "govuk-label--xl"
+  val gdsPageLegend   = "govuk-fieldset__legend--xl"
+  val gdsPageHeading  = "govuk-heading-l"
```

Comment:
I think we might have to retain to govuk-heading-xl because the GRS pages use govuk-heading-xl, so to main consistency across pages we might have to change in the page we are using govuk-heading-l

Reply:
I think we agreed that since our prototype uses l (rather than xl) then we should. It sounds like the best way forward is for GRS to permit some customisation in this area. I think Joel said he was going to pick that up with them.